src/resources/UsageAnalytics/Read/Filters/FiltersInterfaces.ts

```diff
@@ -0,0 +1,35 @@
+export interface FilterModel {
+    /** The display name of the filter. */
+    displayName: string;
+
+    /** The actual value of the filter. */
+    value: string;
+}
+
+export interface FilterResponse {
+    /** The filter id. */
+    id: string;
+
+    /** The filter display name. */
+    displayName: string;
+
+    /** The filter account. */
+    account: string;
+
+    /** The filter type. Can be 'permissions' or 'reporting'. */
+    type: string;
+
+    /** The actual filter. */
+    value: string;
+}
+
+export type CreateReportingFilterModel = FilterModel;
+export type CreateReportingFilterResponse = Pick<FilterResponse, 'id'>;
+
+export type UpdateReportingFilterModel = CreateReportingFilterModel;
+export type UpdateReportingFilterResponse = CreateReportingFilterResponse;
+
+export interface ListFiltersResponse {
+    /** A list of filters */
```

Comment:
Nitpick here, but I would maybe list the properties that will be return per filters (which are `displayValue` and `value`).

Reply:
`FilterModel` provides more information on those fields, and including that here could lead to misinformation in the future if `FilterModel` changes. I have been following the descriptions available in the Swagger docs so far, which usually don't include information about fields in nested properties.

Reply:
Good point!